packages/edit-site/src/components/editor/use-adapt-editor-to-canvas.js

```diff
@@ -17,6 +22,8 @@ export function useAdaptEditorToCanvas( canvas ) {
 	} = useDispatch( editorStore );
 	const { get: getPreference } = useSelect( preferencesStore );
 	const registry = useRegistry();
+	const { resetZoomLevel } = unlock( useDispatch( blockEditorStore ) );
+	const { isZoomOut } = unlock( select( blockEditorStore ) );
```

Comment:
The components shouldn't use global `select` it doesn't react to store changes.

```suggestion
	const { isZoomOut } = unlock( useSelect( blockEditorStore ) );
```